pkg/cmd/grafana-cli/commands/list_upgradeable_command.go

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import (
+	"errors"
+
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/services"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/utils"
+	"github.com/hashicorp/go-version"
+)
+
+func (cmd Command) listUpgradeableCommand(c utils.CommandLine) error {
+	pluginsDir := c.PluginDirectory()
+
+	localPlugins := services.GetLocalPlugins(pluginsDir)
+
+	remotePlugins, err := cmd.Client.ListAllPlugins(c.String("repo"))
+	if err != nil {
+		return err
+	}
+
+	anythingToUpgrade := false
+
+	for _, localPlugin := range localPlugins {
+		for _, remotePlugin := range remotePlugins.Plugins {
+			if localPlugin.ID != remotePlugin.ID {
+				continue
+			}
+			if shouldUpgrade(localPlugin.Info.Version, &remotePlugin) {
+				latest := latestSupportedVersion(&remotePlugin)
```

Comment:
Not using `remotePlugin := p` (which was suggested) creates the following warnings/errors:

Should I just add the temporary variable back @wbrowne ?

```
pkg/cmd/grafana-cli/commands/list_upgradeable_command.go:30:47: G601: Implicit memory aliasing in for loop. (gosec)
                        if shouldUpgrade(localPlugin.Info.Version, &remotePlugin) {
                                                                   ^
pkg/cmd/grafana-cli/commands/list_upgradeable_command.go:31:38: G601: Implicit memory aliasing in for loop. (gosec)
                                latest := latestSupportedVersion(&remotePlugin)
```